satpy/readers/fci_l1c_nc.py

```diff
@@ -186,7 +186,7 @@ def __init__(self, filename, filename_info, filetype_info):
         """Initialize file handler."""
         super().__init__(filename, filename_info,
                          filetype_info,
-                         cache_var_size=10000,
+                         cache_var_size=0,
```

Comment:
Does this mean that the cache will not be use per default?

Reply:
I believe this is to stop the base class to cache anything. Only the variables wrapped in `get_and_cache_npxr()` calls will be cached for this file handler.